samples/getMetadata.js

```diff
@@ -41,34 +41,44 @@ function main(bucketName = 'my-bucket', filename = 'test.txt') {
       .file(filename)
       .getMetadata();
 
-    console.log(`File: ${metadata.name}`);
     console.log(`Bucket: ${metadata.bucket}`);
-    console.log(`Storage class: ${metadata.storageClass}`);
-    console.log(`Self link: ${metadata.selfLink}`);
-    console.log(`ID: ${metadata.id}`);
-    console.log(`Size: ${metadata.size}`);
-    console.log(`Updated: ${metadata.updated}`);
+    console.log(`CacheControl: ${metadata.cacheControl}`);
```

Comment:
Not required, but throwing this out there ...  
```js
for (const key in metadata) {
  console.log(`${key}: ${metadata[key]}`);
}
```

Reply:
Haha, I like that as well. The parent sample calls them out line by line, which I assume is to show the sample reader what they'll get without requiring them to run it. It's definitely not the most future-proof plan, but assuming these properties won't be removed, the annoyance of typing them all out is already spent :)